fahrschein/src/main/java/org/zalando/fahrschein/CursorOffsetCommitException.java

```diff
@@ -0,0 +1,61 @@
+package org.zalando.fahrschein;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import org.zalando.fahrschein.domain.Cursor;
+
+@SuppressWarnings("serial")
+public class CursorOffsetCommitException extends IOException {
```

Comment:
It makes sense change the name to `CommitException` (or `CursorCommitException`), as right now class name is incorrectly representing what have happened.